Neos.Flow/Classes/Persistence/Doctrine/DataTypes/JsonArrayType.php

```diff
@@ -46,15 +43,11 @@ public function getName(): string
      */
     public function getSQLDeclaration(array $column, AbstractPlatform $platform): string
     {
-        if ($platform instanceof PostgreSQL94Platform) {
-            return 'jsonb';
-        }
-
```

Comment:
This change should probably be reverted – even if it might not be required for dbal 3.x